### DISCUSSION TITLE
Add more documentation on Logic Analyzer (kefir collection)

### DISCUSSION THREAD

**Discussion by drtrigon** [FPGAwars/icestudio#855] [Ideas]
The Logic Analyzer examples lack on documentation and use all FT245 for data transmission. Some boards like e.g. the Alhambra 2 do not have a FT245 link (e.g. the FTDI chip is not wired properly). It would be very helpful if the examples could be extended with:

* documentation of the logic analyzer, like e.g. what format is the data sent? is it just streaming? or does it need control commands? what are these commands?
* explanation on how to receive and process the data (through FT245), e.g. what additional software is needed? can that be done on the PC side? or does it need additional hardware, like an arduino?
* explanation on how to change the async FT245 to RS232/USB? what baudrate? etc.?

As the Alhambra 2 lacks in FT245 capability I would like to modify the async code the work with RS232/USB. Any help for that would be great too.

**Comment by Harvie** [comment]
Have you tried sigrok/pulseview software? It can use FTDI chip as 8ch logic analyzer (if you wire signal to correct pins)
